generator/common.ts

```diff
@@ -103,7 +103,7 @@ export const CHAIN_TO_CHAIN_ID = {
   Avalanche: avalanche.id,
   Metis: metis.id,
   Base: base.id,
-  Bsc: bsc.id,
+  Bnb: bsc.id,
```

Comment:
Should this be bnb.id?

Reply:
the .id is given by viem so we can not change